packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -74,46 +74,61 @@ contract L2Genesis is Script, Artifacts {
         0xa0Ee7A142d267C1f36714E4a8F75612F20a79720
     ];
 
-    string internal outfile;
+    function name() public pure override returns (string memory) {
+        return "L2Genesis";
+    }
 
     /// @dev Reads the deploy config, sets `outfile` which is where the `vm.dumpState` will be saved to, and
     ///      loads in the addresses for the L1 contract deployments.
     function setUp() public override {
```

Comment:
you can technically delete this `setUp` as it does nothing in addition to the `super.setUp`